build/output.go

```diff
@@ -0,0 +1,86 @@
+package build
+
+import (
+	"encoding/csv"
+	"os"
+	"strings"
+
+	"github.com/moby/buildkit/client"
+	"github.com/pkg/errors"
+)
+
+func ParseOutputs(inp []string) ([]client.ExportEntry, error) {
```

Comment:
can we import https://github.com/moby/buildkit/tree/master/cmd/buildctl/build for deduplication?

Reply:
Generally yes, but can probably be a follow-up when things are more stabilized. Looking at it, I do see that there are slight incompatibilities both ways atm that we should definitely fix.